bc_obps/rls/utils/grant.py

```diff
@@ -18,7 +19,11 @@ class RlsGrant:
     """
 
     def __init__(
-        self, role: RlsRoles, grants: List[RlsOperations], table: RegistrationTableNames, schema: Schemas = Schemas.ERC
+        self,
+        role: RlsRoles,
+        grants: List[RlsOperations],
+        table: RegistrationTableNames | ReportingTableNames,
```

Comment:
This makes the Rls app dependent on the registration and reporting apps - effectively a circular dependency in our app tree.
I'd suggest type `str` instead - since this class can build a grant for any table